pkg/performanceprofile/controller/performanceprofile/components/manifestset/manifestset.go

```diff
@@ -78,7 +78,8 @@ func GetNewComponents(profile *performancev2.PerformanceProfile, opts *component
 	}
 
 	nodeConfig := node.NewNodeConfig(apiconfigv1.CgroupModeV1)
-	runtimeClass := runtimeclass.New(profile, machineconfig.HighPerformanceRuntime)
+	rtName := runtimeclass.BuildRuntimeClassName(profile.Name)
```

Comment:
is this a refactor or a needed change for the PR ?

Reply:
+1

Reply:
In general I have tried to reuse as much code as possible so any change in the performance_profile controller's way of doing things will affect also the PP hypershift controller.
In this particular case [here](https://github.com/jlojosnegros/cluster-node-tuning-operator/blob/db3912baafe05eab080b65628c4d5dfc23e68958/pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go#L283) I need to get the rtclass name so I extracted that to a functions of its own for reuse.